AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -88,6 +90,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * An ASLayoutSpec subclass that can wrap a ASLayoutable and calculates the layout of the child.
+ */
+@interface ASWrapperLayoutSpec : ASLayoutSpec
```

Comment:
What's the point of a wrapper layout spec if you can just set the size of a node or spec directly? Could you give an example of what it might be useful for in the comment?